templateflow/api.py

```diff
@@ -292,7 +292,7 @@ def _s3_get(filepath):
     import requests
 
     path = str(filepath.relative_to(TF_LAYOUT.root))
```

Comment:
I think the better fix would be:

```suggestion
    path = filepath.relative_to(TF_LAYOUT.root).as_posix()
```